stable/prometheus/values.yaml

```diff
@@ -1,8 +1,14 @@
+rbac:
+  create: false
+
 alertmanager:
   ## If false, alertmanager will not be installed
   ##
   enabled: true
 
+  # Defines the serviceAccountName to associate with a role when rbac is enabled.
+  serviceAccountName: default
```

Comment:
That comment above is confusing, then. It says, "when [RBAC] is enabled," which I took to mean "when this chart plays along with RBAC."